src/gui/src/MainWindow.cpp

```diff
@@ -443,19 +447,20 @@ void MainWindow::checkConnected(const QString& line)
 
 void MainWindow::checkFingerprint(const QString& line)
 {
-    QRegExp fingerprintRegex(".*peer fingerprint \\(SHA1\\): ([A-F0-9:]+) \\(SHA256\\): ([A-F0-9:]+)");
-    if (!fingerprintRegex.exactMatch(line)) {
+    QRegularExpression fingerprintRegex(QRegularExpression::anchoredPattern(".*peer fingerprint \\(SHA1\\): ([A-F0-9:]+) \\(SHA256\\): ([A-F0-9:]+)"));
```

Comment:
Finally found how to write suggestions! Here’s my previous comment and the contents of  #1667 in an easier form:

```suggestion
    QRegularExpression fingerprintRegex("peer fingerprint \\(SHA1\\): ([A-F0-9:]+) \\(SHA256\\): ([A-F0-9:]+)$");
```

Essentially avoiding the function that just wraps a pattern in `^...$` by adding the useful end-of-line marker ourselves.